packages/esm-patient-forms-app/src/hooks/use-forms.ts

```diff
@@ -19,22 +19,28 @@ import {
 } from '../constants';
 import { isValidOfflineFormEncounter } from '../offline-forms/offline-form-helpers';
 
-export function useFormEncounters(cachedOfflineFormsOnly = false, patientUuid: string = '', visitUuid: string = '') {
+function useCustomFormsUrl(patientUuid: string, visitUuid: string) {
   const { customFormsUrl, showHtmlFormEntryForms } = useConfig<ConfigObject>();
   const hasCustomFormsUrl = Boolean(customFormsUrl);
-  let url = hasCustomFormsUrl
-    ? customFormsUrl.indexOf('?') === -1
-      ? `${customFormsUrl}?patientUuid=${patientUuid}`
-      : customFormsUrl
-    : showHtmlFormEntryForms
-      ? formEncounterUrl
-      : formEncounterUrlPoc;
-  url = interpolateUrl(url, {
```

Comment:
Defer to others on whether we can remove compatibility for custom urls with no parameters.  I do think it's nicer to remove this, but might need to make this a (breaking) change?